pgp/keysource.go

```diff
@@ -0,0 +1,202 @@
+package pgp
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"github.com/howeyc/gopass"
+	gpgagent "go.mozilla.org/gopgagent"
+	"golang.org/x/crypto/openpgp"
+	"golang.org/x/crypto/openpgp/armor"
+	"io/ioutil"
+	"os"
+	"os/user"
+	"path"
+	"strings"
+	"time"
+)
+
+const DateFormat = "2006-01-02T15:04:05Z"
+
+type GPGMasterKey struct {
```

Comment:
Actually, I think it should just be `MasterKey`. `pgp.PGPMasterKey` is a bit redundant, but I agree in that it should have been PGPMasterKey.